action.yml

```diff
@@ -72,6 +75,8 @@ runs:
         source "${{ github.action_path }}/setup/venv.bash"
 
         python "${{ github.action_path }}/action.py" "${{ inputs.inputs }}"
+
+        echo "summary-raw=$(cat $GITHUB_STEP_SUMMARY)" >> $GITHUB_ENV
       env:
         GHA_PIP_AUDIT_SUMMARY: "${{ inputs.summary }}"
```

Comment:
NB: A user who passes `summary: false` will get an empty `summary-raw`, which may or may not be surprising. 

We should either document this in the README ("summary-raw will be empty if you disable summaries") or work around it.

Reply:
good observation - added a check for this during the `pip-audit` step